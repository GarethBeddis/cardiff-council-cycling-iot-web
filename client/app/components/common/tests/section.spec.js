import React from 'react'
import { shallow } from 'enzyme'

import Section from '../section'

// Prepare a mock child
const MockChild = () => {
  return (
    <div className='child-div' />
  )
}

describe('Section Component', () => {
  // General Tests
  it('Renders without crashing', () => {
    shallow(
      <Section title='Test'>
        <MockChild />
      </Section>
    )
  })
  // Snapshot Tests
  describe('Snapshots', () => {
    it('Section matches Snapshot', () => {
      let wrapper = shallow(
        <Section title='Test'>
          <MockChild />
        </Section>
      )
      expect(wrapper).toMatchSnapshot()
    })
  })
})
