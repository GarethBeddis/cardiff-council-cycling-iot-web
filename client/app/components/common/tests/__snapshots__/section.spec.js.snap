// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Component Snapshots Section matches Snapshot 1`] = `
<section>
  <h1>
    Test
  </h1>
  <MockChild />
</section>
`;
